etc/behat/services/contexts/ui.xml

```diff
@@ -15,6 +15,7 @@
     <parameters>
         <parameter key="sylius.behat.context.ui.admin.managing_countries.class">Sylius\Behat\Context\Ui\Admin\ManagingCountriesContext</parameter>
         <parameter key="sylius.behat.context.ui.admin.managing_locale.class">Sylius\Behat\Context\Ui\Admin\ManagingLocalesContext</parameter>
+        <parameter key="sylius.behat.context.ui.admin.managing_payment_method.class">Sylius\Behat\Context\Ui\Admin\ManagingPaymentMethodContext</parameter>
```

Comment:
`sylius.behat.context.ui.admin.managing_payment_method.class` -> `sylius.behat.context.ui.admin.managing_payment_methods.class` and accordingly `ManagingPaymentMethodsContext`